junit-jupiter-engine/src/test/java/org/junit/jupiter/engine/execution/ParameterResolutionUtilsTests.java

```diff
@@ -93,23 +92,23 @@ void constructorInjectionWithMissingResolver() {
 	}
 
 	@Test
-	void invokingMethodsWithoutParameterDoesNotDependOnParameterResolvers() {
+	void resolvingArgumentsForMethodsWithoutParameterDoesNotDependOnParameterResolvers() {
 		testMethodWithNoParameters();
 		throwDuringParameterResolution(new RuntimeException("boom!"));
 
-		invokeMethod();
+		Object[] arguments = resolveMethodParameters();
 
-		verify(instance).noParameter();
+		assertThat(arguments).hasSize(0);
 	}
 
 	@Test
 	void resolveArgumentsViaParameterResolver() {
 		testMethodWithASingleStringParameter();
 		thereIsAParameterResolverThatResolvesTheParameterTo("argument");
 
-		invokeMethod();
+		Object[] arguments = resolveMethodParameters();
 
-		verify(instance).singleStringParameter("argument");
+		assertThat(arguments).hasSize(1).containsExactly("argument");
```

Comment:
```suggestion
		assertThat(arguments).containsExactly("argument");
```

`hasSize()` is already covered by `containsExactly()`